be/src/exprs/cast_expr.cpp

```diff
@@ -1127,6 +1134,43 @@ class VectorizedCastExpr final : public Expr {
         }
         return result_column;
     };
+
+    bool is_compilable() const override {
+        return !AllowThrowException && IRHelper::support_jit(FromType) && IRHelper::support_jit(ToType);
+    }
+
+    StatusOr<LLVMDatum> generate_ir_impl(ExprContext* context, const llvm::Module& module, llvm::IRBuilder<>& b,
+                                         const std::vector<LLVMDatum>& datums) const override {
+        auto* l = datums[0].value;
+
+        if constexpr (FromType == TYPE_JSON || ToType == TYPE_JSON) {
+            return Status::NotSupported("JIT casting does not support JSON");
+        } else if constexpr (lt_is_decimal<FromType> || lt_is_decimal<ToType>) {
+            return Status::NotSupported("JIT casting does not support decimal");
```

Comment:
the `is_compilable()` can't rejet such cases?

Reply:
This is for the following`std::numeric_limits<RunTimeCppType<ToType>>::max()` can be compiled.